server/README-ECLIPSE.md

```diff
@@ -0,0 +1,40 @@
+Eclipse Installation
+--------------------
+
+The installation of the various plug-ins we use for development,
+code quality, etc., is straightforward, but is nowhere near as automated
+as we would like. To install an Eclipse distribution with all the required
+tools and get the project ready for development, do the following:
+
+  1. Download the Eclipse Oxygen installer for your platform from
+  http://www.eclipse.org/oxygen/.
+  2. Run the installer and install the "Eclipse IDE for Java Developers"
+  (the first option on the list of distributions to install).
```

Comment:
I don't find clear or convincing installation documentation there. Do we just run `eclipse-inst`? Here's all I see.
Clicking "Download" yields a .tar.gz file.
`tar zxf eclipse-inst-linux64.tar.gz` yields an `eclipse-installer` directory.
That directory has a `readme/readme_eclipse.html` which gives no obvious hints about what to do next.
I'm guessing I should run the executable file `eclipse-inst`, but running random undocumented executables off the internet with only a grandiose sha512 hash (`2bbb38475b3f3b13b9c6c5db5895fe25ce24b4a8df6a24c6822a348a9652dc8df49ca19152495f34156c39022b8c2ed6b5bb1e84db3edc751302d73bdc1d56da  eclipse-inst-linux64.tar.gz`, FWIW) is not my idea of a secure plan.
Where is a PGP signature, or a proper signed package? Yeah, this is feedback for eclipse, but ....
This looks like a useful link: [How to install Eclipse using its installer - Ask Ubuntu](https://askubuntu.com/questions/695382/how-to-install-eclipse-using-its-installer)